### DISCUSSION TITLE
Is it possible to delete an element by its index?

### DISCUSSION THREAD

**Discussion by galnir** [prisma/prisma#12629] [Q&A] [answered]
These are my models (also should mention I'm using PostgreSQL):
<img width="1028" alt="Screen Shot 2022-04-01 at 19 00 09" src="https://user-images.githubusercontent.com/36937694/161300200-e8b9a048-c0f8-4531-a782-1dca95d7b59c.png">

Is it possible to delete a Song from a Playlist using the song's index in the playlist?

**Comment by austincrim** [comment, marked as answer]
Hey @galnir 👋 ,

It's not possible to target list fields by index in Prisma but you could use [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryRaw) for this.

Feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if you would like to see support for this in the future!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
